deploy.sh

```diff
@@ -49,50 +58,50 @@ function deploytotest {
-	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@test.schul-cloud.org /usr/bin/docker service update --force --image schulcloud/schulcloud-server:$DOCKERTAG test-schul-cloud_server
+	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i .build/travis_rsa travis@test.schul-cloud.org schulcloud/schulcloud-server:$DOCKERTAG test-schul-cloud_server
 }
 
 function deploytoprods {
 	# Deploys new masters on the instances brandenburg, open, demo
 	# compose-files are distributed via Ansible, many different secrets, Mongo_URIs etc.
 
 	# copy config-file to server and execute mit travis_rsa
-	chmod 600 travis_rsa
+	chmod 600 .build/travis_rsa
 
 	# brandenburg
-	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@open.schul-cloud.org /usr/bin/docker service update --force --image schulcloud/schulcloud-server:latest brabu_server
+	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i .build/travis_rsa travis@open.schul-cloud.org schulcloud/schulcloud-server:latest brabu_server
 	# open
-	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@open.schul-cloud.org /usr/bin/docker service update --force --image schulcloud/schulcloud-server:latest open_server
+	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i .build/travis_rsa travis@open.schul-cloud.org schulcloud/schulcloud-server:latest open_server
 	# thueringen
-	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@schulcloud-thueringen.de /usr/bin/docker service update --force --image schulcloud/schulcloud-server:latest thueringen_server
+	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i .build/travis_rsa travis@schulcloud-thueringen.de schulcloud/schulcloud-server:latest thueringen_server
 	# demo
-	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@demo.schul-cloud.org /usr/bin/docker service update --force --image schulcloud/schulcloud-server:latest demo_server
+	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i .build/travis_rsa travis@demo.schul-cloud.org schulcloud/schulcloud-server:latest demo_server
 }
 
 function deploytostaging {
 	# Deploys new masters on the instances brandenburg, open, demo
 	# compose-files are distributed via Ansible, many different secrets, Mongo_URIs etc.
 
 	# copy config-file to server and execute mit travis_rsa
-	chmod 600 travis_rsa
+	chmod 600 .build/travis_rsa
 
 	# staging
-	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@staging.schul-cloud.org /usr/bin/docker service update --force --image schulcloud/schulcloud-server:$DOCKERTAG staging_server
+	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i .build/travis_rsa travis@staging.schul-cloud.org schulcloud/schulcloud-server:$DOCKERTAG staging_server
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045775)